{"executionId": "312acbf9-bde4-41e5-ac26-40652073be9e", "createdAt": "2024-03-30T18:17:15.165708Z", "event": "form-message", "payload": {"messageType": "execution:start", "messagePayload": {"type": "execution:start", "params": {}}}, "sequence": 1}
{"executionId": "312acbf9-bde4-41e5-ac26-40652073be9e", "createdAt": "2024-03-30T18:17:15.166284Z", "event": "form-message", "payload": {"messageType": "execution:started", "messagePayload": {"executionId": "312acbf9-bde4-41e5-ac26-40652073be9e"}}, "sequence": 2}
{"executionId": "312acbf9-bde4-41e5-ac26-40652073be9e", "createdAt": "2024-03-30T18:17:15.170565Z", "event": "form-message", "payload": {"messageType": "form:mount-page", "messagePayload": {"widgets": [{"type": "text-input", "key": "\ud83d\udc4b Hello there! What is your name?", "label": "\ud83d\udc4b Hello there! What is your name?", "value": "", "placeholder": "", "required": true, "hint": null, "fullWidth": false, "mask": null, "disabled": false, "errors": []}], "actions": ["i18n_next_action"], "endProgram": false, "reactivePollingInterval": 0, "steps": null}}, "sequence": 3}
{"executionId": "312acbf9-bde4-41e5-ac26-40652073be9e", "createdAt": "2024-03-30T18:17:26.914222Z", "event": "form-message", "payload": {"messageType": "connection-closed", "messagePayload": {"message": "[ERROR] Unhandled connection closed with code 4000: FORM_EDITOR_RESTART\n", "reason": 4000}}, "sequence": 4}
{"executionId": "312acbf9-bde4-41e5-ac26-40652073be9e", "createdAt": "2024-03-30T18:17:26.914896Z", "event": "unhandled-exception", "payload": {"text": "Traceback (most recent call last):\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/form_execution.py\", line 112, in recv\n    str_data = self._connection.receive()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/simple_websocket/ws.py\", line 104, in receive\n    raise ConnectionClosed(self.close_reason, self.close_message)\nsimple_websocket.errors.ConnectionClosed: Connection closed: 4000 FORM_EDITOR_RESTART\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/execution.py\", line 236, in run\n    import_as_new(self.stage.file)\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/modules.py\", line 21, in import_as_new\n    module_spec.loader.exec_module(module)\n  File \"<frozen importlib._bootstrap_external>\", line 883, in exec_module\n  File \"<frozen importlib._bootstrap>\", line 241, in _call_with_frames_removed\n  File \"/Users/raj/nanu-project/product_name.py\", line 6, in <module>\n    name = af.read(\"\ud83d\udc4b Hello there! What is your name?\")\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/interface/sdk/forms/generated/inputs.py\", line 684, in read\n    return get_single_value(Page().read(label, **kwargs).run(button_text))\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/interface/sdk/forms/page.py\", line 104, in run\n    response: Dict = self.__handle_page_user_events(validate_func=validate)\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/interface/sdk/forms/page.py\", line 113, in __handle_page_user_events\n    response = receive()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/interface/sdk/forms/connection.py\", line 12, in receive\n    return execution.receive()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/form_execution.py\", line 140, in receive\n    type, data = self.recv()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/form_execution.py\", line 114, in recv\n    self.close()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/form_execution.py\", line 123, in close\n    self._connection.close(message=reason)\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/simple_websocket/ws.py\", line 115, in close\n    raise ConnectionClosed(self.close_reason, self.close_message)\nsimple_websocket.errors.ConnectionClosed: Connection closed: 4000 FORM_EDITOR_RESTART\n"}, "sequence": 5}
{"executionId": "312acbf9-bde4-41e5-ac26-40652073be9e", "createdAt": "2024-03-30T18:17:26.915326Z", "event": "form-message", "payload": {"messageType": "execution:stderr", "messagePayload": {"log": "Traceback (most recent call last):\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/form_execution.py\", line 112, in recv\n    str_data = self._connection.receive()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/simple_websocket/ws.py\", line 104, in receive\n    raise ConnectionClosed(self.close_reason, self.close_message)\nsimple_websocket.errors.ConnectionClosed: Connection closed: 4000 FORM_EDITOR_RESTART\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/execution.py\", line 236, in run\n    import_as_new(self.stage.file)\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/modules.py\", line 21, in import_as_new\n    module_spec.loader.exec_module(module)\n  File \"<frozen importlib._bootstrap_external>\", line 883, in exec_module\n  File \"<frozen importlib._bootstrap>\", line 241, in _call_with_frames_removed\n  File \"/Users/raj/nanu-project/product_name.py\", line 6, in <module>\n    name = af.read(\"\ud83d\udc4b Hello there! What is your name?\")\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/interface/sdk/forms/generated/inputs.py\", line 684, in read\n    return get_single_value(Page().read(label, **kwargs).run(button_text))\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/interface/sdk/forms/page.py\", line 104, in run\n    response: Dict = self.__handle_page_user_events(validate_func=validate)\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/interface/sdk/forms/page.py\", line 113, in __handle_page_user_events\n    response = receive()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/interface/sdk/forms/connection.py\", line 12, in receive\n    return execution.receive()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/form_execution.py\", line 140, in receive\n    type, data = self.recv()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/form_execution.py\", line 114, in recv\n    self.close()\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/abstra_internals/execution/form_execution.py\", line 123, in close\n    self._connection.close(message=reason)\n  File \"/Users/raj/nanu-project/.venv/lib/python3.10/site-packages/simple_websocket/ws.py\", line 115, in close\n    raise ConnectionClosed(self.close_reason, self.close_message)\nsimple_websocket.errors.ConnectionClosed: Connection closed: 4000 FORM_EDITOR_RESTART\n"}}, "sequence": 6}
